### ISSUE TITLE
ActiveField::staticControl inputOptions

### ISSUE THREAD

**Issue by dicrtarasov** [yiisoft/yii2-bootstrap4#168]
Missing inputOptions in \yii\bootstrap4\ActiveField::staticControl

```php
    public function staticControl($options = [])
    {
       +   $options = array_merge($this->inputOptions, $options);
        $this->adjustLabelFor($options);
        $this->parts['{input}'] = Html::activeStaticControl($this->model, $this->attribute, $options);
        return $this;
    }
```
Yii version - 2.0.23
yiisoft/yii2-bootstrap4 - 2.0.6

**Comment by simialbi** [comment]
I don't think it makes sense to copy the general "input options" to static control too. It has other classes and properties:
https://getbootstrap.com/docs/4.3/components/forms/#readonly-plain-text

**Comment by dicrtarasov** [comment]
> I don't think it makes sense to copy the general "input options" to static control too. It has other 

For what other purposes and scenarios is this field intended and can be used? ActiveField renderer only by one of methods - textField or staticField or anothe. ActiveField instance not using many times with different input rendering. So, all it data intended to be used just for one purpose - rendering field. I'm usng ActiveField in my library and all workng fine except the staticControl, which, unlike other methods, loses and does not take into account the parameters. This is an obvious bug.

**Comment by simialbi** [comment]
No the static control is not like other fields. It's not just the readonly property, it has other classes and just exists in bootstrap library cause it just exists in bootstrap framework. It makes not sense to copy all the options from default inputs and have to unset them in the `staticControl` method again...
